### ISSUE TITLE
Documentation fix?

### ISSUE THREAD

**Issue by firescript** [corcel/corcel#264]
I think these two should be switched no? Currently its:
```So, now you can fetch database data:

$posts = App\Post::all(); // using the 'wordpress' connection
$posts = Corcel\Post::all(); // using the 'default' Laravel connection
```
Should be:
```
$posts = App\Post::all(); // using the 'default' Laravel connection
$posts = Corcel\Post::all(); // using the 'wordpress' connection
```

**Comment by jgrossi** [comment]
Yep, you're right.

Actually I changed the basic code to look for `wordpress` or `corcel` connection names as well. If you are using Laravel and you only have `default` connection, Corcel would use it. But if you have a `wordpress` or `corcel` connection it would be used by Corcel.

The goal using `App\Post` is when using a different connection, like `foobar`, so you would have to create your own `Post` class (usually in the `App` namespace) and set the `$connection` attribute to the desired connection name.

I'm going to think how to better include this in the docs. If you have any idea please send a PR.

Cheers, JG.

**Comment by firescript** [comment]
So for people who want to extend the `Corcel\Post` model into their own `App\Post` class they would use the 'protected $connection = "wordpress"' to give their model Corcel ability. But if you're content with just using `Corcel\Post` as your model in your controller to get data, you really don't even need a new Post model. 

I'm currently just doing `use Corcel\Post` in my PageController to pull what I need for me. I wonder if its worth explaining that.

I just remember reading that and feeling confused at my first time through the docs. The other quick thing I noticed was that I had to find your blog post on where to install WordPress and how to properly set it up, instead of it just being here in the docs.

I'll work on a PR for a docs update for you to review.
